### PR TITLE
Time Series Forecaster basic Tutorial

### PR DESCRIPTION
**Pull request by abgese**
### Which issue(s) does this Pull Request fix?

resolves #000

### CODE REVIEW

File: docs/py/timeseries_forecaster.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+data_x_val = validation_data[['CO(GT)', 'PT08.S1(CO)', 'NMHC(GT)', 'C6H6(GT)',
+                              'PT08.S2(NMHC)', 'NOx(GT)', 'PT08.S3(NOx)', 'NO2(GT)',
+                              'PT08.S4(NO2)', 'PT08.S5(O3)', 'T', 'RH']].astype(
+                                  'float64')
+
+data_y = data_train['AH'].astype('float64')
+
+data_y_val = validation_data['AH'].astype('float64')
+
```
Comment (haifeng-jin) [inline docs/py/timeseries_forecaster.py:45]: It is good to print the shape of the data here.

File: docs/py/timeseries_forecaster.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    predict_from=predict_from,
+    predict_until=predict_until,
+    max_trials=1,
+    objective='val_loss'
+)
+
+clf.fit(x=data_x, y=data_y, validation_data=(data_x_val, data_y_val), batch_size=30,
+        epochs=10)
+
+predictions = clf.predict(data_x_val)
```
Comment (haifeng-jin) [inline docs/py/timeseries_forecaster.py:67]: print the shape of the `predictions`.
Also call evaluation to see and print the result with a readable message `'...'.format()`

**Review by haifeng-jin — CHANGES_REQUESTED** [review]
Thanks for the PR! Please sign the Google CLA following the instructions commented by the bot on this PR. Also, try to run the formatted to fix the CI.

File: docs/py/timeseries_forecaster.py (hunk 2 above)
Comment (iamianM) [reply docs/py/timeseries_forecaster.py:67]: my predictions array looks like this: `array([], shape=(0, 1), dtype=float32)`
Have you encountered this issue before?

Comment (abgese) [reply docs/py/timeseries_forecaster.py:67]: Figured out the issue here. In the predict function, the y_pred is sliced to only output the predictions from "train_len + predict_from" to "train_len + predict_until". Therefore, if we use data_x_val(which has length less than the train_len) the function will return an empty array. I'll try to figure out what makes the most sense here.

Note: Another thing that might cause some issues with the slicing is that the length of the y_pred before slicing is equal to "test_len - look_back + 1". Sp a very large lookback might result in an empty prediction array again.

**Comment by abgese** [comment]
@googlebot I signed it!

File: docs/py/timeseries_forecaster.py (hunk 2 above)
Comment (abgese) [reply docs/py/timeseries_forecaster.py]: So based on the how the api is expected to be used we are currently going to enforce, that the predict data needs to contain the original train data time steps along with the time steps for which the prediction needs to be done.

**Review by stancsz — APPROVED** [review]
I ran your branch & it seem functional, thanks for the tutorial

File: docs/py/timeseries_forecaster.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    predict_from=predict_from,
+    predict_until=predict_until,
+    max_trials=1,
+    objective='val_loss'
+)
+# Train the TimeSeriesForecaster with train data
+clf.fit(x=data_x, y=data_y, validation_data=(data_x_val, data_y_val), batch_size=32,
+        epochs=10)
+# Predict with the best model(includes original training data).
+predictions = clf.predict(data_x_test)
```
Comment (stancsz) [inline docs/py/timeseries_forecaster.py:75]: the returned predictions are min-max scaled, not of the original scale. Is this intended?

**Review by stancsz — CHANGES_REQUESTED** [review]

File: docs/py/timeseries_forecaster.py (hunk 3 above)
Comment (abgese) [reply docs/py/timeseries_forecaster.py:75]: I'm sorry could you clarify what you mean?

Comment (stancsz) [reply docs/py/timeseries_forecaster.py:75]: image

image
@abgese  apologies, I have a use case that resulted in numbers like so. I thought they are min-max scaled, but really they're not. I think I must be using the code incorrectly. 
👀 & trying to find out what I'm doing wrong. 

please ignore this and let me move this conversation elsewhere.

File: docs/py/timeseries_forecaster.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+print(data_x.shape)  # (6549, 12)
+print(data_y.shape)  # (6549,)
+
+"""
+The second step is to run the [TimeSeriesForecaster](/time_series_forecaster).
+As a quick demo, we set epochs to 10.
+You can also leave the epochs unspecified for an adaptive number of epochs.
+"""
+import autokeras as ak
```
Comment (stancsz) [inline docs/py/timeseries_forecaster.py:59]:
```
E402 module level import not at top of file
```
the import of `autokeras` should be place on top of the file
context:
https://www.flake8rules.com/rules/E402.html

**Review by stancsz — CHANGES_REQUESTED** [review]
```
E402 module level import not at top of file
```
the import of `autokeras` should be place on top of the file
context:
https://www.flake8rules.com/rules/E402.html

**Review by stancsz — APPROVED** [review]

**Pull request merged**